frontend/src/employee-frontend/components/document-templates/template-editor/DocumentTemplatesPage.tsx

```diff
@@ -95,10 +107,15 @@ const TemplateRow = React.memo(function TemplateRow({
   const { mutateAsync: deleteDocumentTemplate } = useMutationResult(
     deleteDocumentTemplateMutation
   )
+  const onClickExport = useCallback(() => {
+    window.open(
+      exportDocumentTemplateUrl(template.id, sanitizedFilename(template.name))
```

Comment:
I think it would be better to determine the filename on server-side instead of replying with the filename given in request path